src/status_im/contexts/communities/events.cljs

```diff
@@ -430,16 +333,17 @@
              [:dispatch [:chat/navigate-to-chat chat-id]])]})))
 
 (defn get-revealed-accounts
-  [{:keys [db]} [community-id]]
+  [{:keys [db]} [community-id on-success]]
   (let [{:keys [joined fetching-revealed-accounts]
-         :as   community} (get-in db [:communities community-id])]
-    (when (and community joined (not fetching-revealed-accounts))
+         :as   community} (get-in db [:communities community-id])
+        pending?          (get-in db [:communities/my-pending-requests-to-join community-id])]
+    (when (and community (or pending? joined) (not fetching-revealed-accounts))
```

Comment:
FYI: Revealed addresses should also be fetched when there are pending requests to join.